test/apiv2/10-images.at

```diff
@@ -151,5 +151,6 @@ t DELETE libpod/images/test1:latest 200
 #      actual: {"cause":"no such image","message":"unable to find a name and tag match for busybox in repotags: no such image","response":500}
 #
 #t GET images/get?names=alpine,busybox 200 '[POSIX tar archive]'
+t GET "images/get?names=alpine" 200 '[POSIX tar archive]'
```

Comment:
Why not just uncomment line 153, and get rid of the huge FIXME which is no longer applicable?

Reply:
@edsantiago because I know that if there is only one name it works correctly. If multiple names are provided only first one is taken account.

Reply:
`t GET images/get?names=alpine,busybox 200 '[POSIX tar archive]'` would pass, we should have some better assertion than just content-type.

Reply:
I can change the `TODO`, it should say that this function for now works only with single parameter.

Reply:
Is there simple way to assert content of non-JSON body?

Reply:
hmm also seems that `dockerd` accepts querystring like `?names=alpine&names=busybox`, not comma separated

Reply:
but the documenation says it's comma separated array of string, https://docs.docker.com/engine/api/v1.41/#operation/ImageGet, weird

Reply:
Aaaaaargh! Too many comments to reply to! :smile:

The results are saved in `$WORKDIR/curl.result.out`. You can see `test/apiv2/44-mounts.at` for an example of reading that. Maybe you could run `tar tf` and then do a `like` on that.

I don't know anything about what this endpoint is supposed to do. If the two-argument form doesn't work, please update the comments to say so.

My only request is, please don't leave a confusing obsolete FIXME comment in there. Thank you.

Reply:
Here's a good test:
```bash
t GET images/get?names=$IMAGE 200 '[POSIX tar archive]'
tar_tf=$(tar tf $WORKDIR/curl.result.out)
like "$tar_tf" ".*$iid\.json" "IID of test image found in tarball"
```
(`$iid` is defined way at top). Customizing that to deal with multi-arg form is left as an exercise for the reader.